frontend/src/components/logout/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { h } from "preact";
+
+import { useLocation } from "preact-iso";
+import "./style.css";
+
+const Logout = () => {
+  const location = useLocation();
+  const handleLogout = async () => {
+    try {
+      const token = localStorage.getItem("jwt");
+      if (!token) {
+        console.error("No token found in local storage");
+        return;
+      }
+      await fetch("/api/logout", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${token}`,
+        },
+      });
+
+      localStorage.removeItem("jwt");
```

Comment:
This part should be in a `finally` block. The frontend should remove the jwt regardless of the success of the /logout call

Reply:
Done